sdk/batch/Microsoft.Azure.Batch.Conventions.Files/src/TaskOutputStorage.cs

```diff
@@ -203,14 +169,12 @@ public IEnumerable<OutputFileReference> ListOutputs(TaskOutputKind kind)
         /// <param name="kind">A <see cref="TaskOutputKind"/> representing the category of the output to
         /// retrieve, for example <see cref="TaskOutputKind.TaskOutput"/> or <see cref="TaskOutputKind.TaskLog"/>.</param>
         /// <param name="filePath">The path under which the output was persisted in blob storage.</param>
-        /// <param name="cancellationToken">A <see cref="CancellationToken"/> for controlling the lifetime of the asynchronous operation.</param>
         /// <returns>A reference to the requested file in Azure blob storage.</returns>
-        public async Task<OutputFileReference> GetOutputAsync(
+        public OutputFileReference GetOutput(
```

Comment:
Why was this made synchronous?

Reply:
This calls the underlying _storagePath.GetOutput method which returns an OutputFileReference with the underlying blob reference. In the new SDK, we can simply get the blob reference through the synchronous GetBlobBaseClient:

`Validate.IsNotNullOrEmpty(filePath, nameof(filePath));

            var blob = await _jobOutputContainer.GetBlobReferenceFromServerAsync(BlobName(kind, filePath), null, null, null, cancellationToken).ConfigureAwait(false);
            var blob = _jobOutputContainer.GetBlobBaseClient(BlobName(kind, filePath));`

Looking back though I suppose we could still maintain this as asynchronous.